src/+datools/+statistical/+ensemble/EnF.m

```diff
@@ -1,35 +1,39 @@
 classdef EnF < handle
+    % This is the base class for all statistical methods
```

Comment:
Please follow the guide at https://www.mathworks.com/help/matlab/matlab_prog/create-help-for-classes.html
`% EnF This is a short description` is what we want to be first.